src/main/java/com/cburch/logisim/fpga/hdlgenerator/HDL.java

```diff
@@ -114,4 +120,117 @@ public static String GetZeroVector(int nrOfBits, boolean floatingPinTiedToGround
     return contents.toString();
   }
 
+  public static String getConstantVector(long value, int nrOfBits) {
+    if (HDL.isVHDL()) {
+      if (nrOfBits == 1) return "'" + value + "'";
+      return "std_logic_vector(to_unsigned("
```

Comment:
`value` is a long, but integers in VHDL are limited to 32 bit, so this may cause problems.
(64-bit integer range was introduced in VHDL-2019, but the support for this version is limited.)

Reply:
I agree. Then, we should probably do the conversion to a bit string already on the Java side.

Reply:
Agreed, will take out this one and go completely to ```getConstantBitVector()```

Reply:
Done